discussion/discussion.tex

```diff
@@ -4,15 +4,11 @@ \chapter{Discussion}
 \label{chap:discussion}
 
 This project has covered a lot of implementation involved with {\rust} and {\C} for the {\gecko}, as well as presenting the ecosystem for building and distributing packages written in {\rust}.
```

Comment:
Hva med: `This project has covered a lot of implementation level details involved with binging {\rust} to the {\gecko}, as well as presenting the ecosystem for building and distributing packages written in {\rust}.`